### DISCUSSION TITLE
Problem with Source control

### DISCUSSION THREAD

**Discussion by sergebourlon** [microsoft/vscode-discussions#2435] [Extension Development QnA]
Hi,

I meet problem with "Source control", I have this message 

![Capture](https://github.com/user-attachments/assets/fb80cf66-6ff9-4f87-a7f1-0b24669cb6e7)
![Capture](https://github.com/user-attachments/assets/a9084d5e-6248-4d77-915a-cbe487265ece)

Can you help me ?

I am novice to programmation Python and utilisation VSC

Serge

**Comment by Yasir-oss1** [comment]
Hi Serge,

It seems you're encountering an issue with source control in Visual Studio Code while working on Python. Ensure Git is installed on your system and initialized in your project folder. You can run:

bash
git init
If you still see an error, feel free to share the exact message for more precise help[!](https://halfbirthdaycalculator.online/)

**Reply by olguzzar** [reply]
Hi @sergebourlon, as the above states, you likely do not have a source control provider, like Git, initialized for your repo. The VS Code YouTube channel has some great videos for getting started with source control, like [this one](https://youtu.be/i_23KUAEtUM)!

This forum is specifically for extension authors, so I am going to close this discussion as off topic. Thanks and happy coding!
